operatorconfig/driverconfig/unity/v2.8.0/controller.yaml

```diff
@@ -71,6 +71,16 @@ rules:
   - apiGroups: [""]
     resources: ["persistentvolumeclaims/status"]
     verbs: ["update", "patch"]
+  # Permissions for CSIStorageCapacity
+  - apiGroups: ["storage.k8s.io"]
+    resources: ["csistoragecapacities"]
+    verbs: ["get", "list", "watch", "create", "update", "patch", "delete"]
+  - apiGroups: [""]
+    resources: ["pods"]
```

Comment:
Permissions for pods are already there in Line No 43. Please check

Reply:
done